.github/actions/setup-kind/action.yml

```diff
@@ -32,13 +33,21 @@ inputs:
       The timeout for waiting for the ingress controller to become ready.
       Default is 90 seconds, which should be enough for most cases.
     default: 90s
+  kind-config-override:
+    description: |
+      Optional YAML string to override the default kind configuration passed to
+      kind during cluster creation.
+      See https://kind.sigs.k8s.io/docs/user/configuration/ for details.
```

Comment:
* this description is not current anymore
* maybe better rename to kind-config-path (no need to specify it's an override if it's not mandatory)